src/components/selectYear.js

```diff
@@ -0,0 +1,112 @@
+import React, { PureComponent, PropTypes } from 'react'
+import { connect } from 'react-redux'
+import * as actions from '../actions/actionIndex'
+import reactCSS from 'reactcss'
+
+const styles = reactCSS({
+  'default': {
+    button: {
+      color: 'orange',
+      fontSize: 23,
+      padding: 15,
+      backgroundColor: 'rgba(0, 0, 0, 0)',
+      border: 0,
+      paddingTop: 5,
+    }
+  }
+})
+
+const { object, func} = PropTypes
+
+class SelectYear extends PureComponent {
+
+  state = {
+    historyYear: 1977
+  }
+
+  static PropTypes = {
+    historyYear: object,
+    goToFollowingYear: func,
+    loadHistoryData: func,
+  }
+
+  static defaultProps = {
+    historyYear: 1977,
+  }
+
+  validateYear = (year) => {
+    if (year > 2016) {
+      alert('Please select a year prior to 2017!')
+      return false
+    }
+    else if (year < 1949) {
+      alert('Please select a year later than 1949!')
+      return false
+    }
+    else return true
+  }
+
+  gotoNextYear = () => {    
+    const nextYear = this.state.historyYear + 1
+    const valid = this.validateYear(nextYear)
+    if (valid === true) {
```

Comment:
This could be rewritten as `if (valid) { }`. The `===` comparator is unnecessary. There are a few other places in this file this comment applies (e.g. line 61, 70, 79)